src/detection_win.cpp

```diff
@@ -379,28 +382,28 @@ void BuildInitialDeviceList() {
 	}
 
 	SP_DEVINFO_DATA* pspDevInfoData = (SP_DEVINFO_DATA*) HeapAlloc(GetProcessHeap(), 0, sizeof(SP_DEVINFO_DATA));
-	pspDevInfoData->cbSize = sizeof(SP_DEVINFO_DATA);
-	for(int i=0; DllSetupDiEnumDeviceInfo(hDevInfo, i, pspDevInfoData); i++) {
-		DWORD nSize=0 ;
-		TCHAR buf[MAX_PATH];
+	if (pspDevInfoData) {
+        pspDevInfoData->cbSize = sizeof(SP_DEVINFO_DATA);
+        for(int i=0; DllSetupDiEnumDeviceInfo(hDevInfo, i, pspDevInfoData); i++) {
+            DWORD nSize=0 ;
```

Comment:
Let's opt for keeping the tabs around (mixing in spaces currently)

It's also making the diff hard to read

Reply:
It's my IDE. I just wanted to indent block